breadbox/breadbox/io/hdf5_utils.py

```diff
@@ -155,3 +160,12 @@ def read_hdf5_file(
         if not keep_nans:
             df = df.replace({np.nan: None})
     return df
+
+
+def _validate_read_size(features_length: int, samples_length: int):
+    """
+    Raise a 500 error if estimated size of reading columns and rows exceed 1GB indicating possible memory exhaustion. 
+    TODO: We will need to handle reading large data
+    """
+    if features_length * samples_length * 8 > 1024 * 1024 * 1024:
```

Comment:
How about putting `1024 * 1024 * 1024` into a constant like "MAX_HDF5_READ_IN_BYTES"?  I like doing this for two reasons:
1. it gives the value on the right a clear name
2. 

Then you can also mock the value without mocking the validation function (which allows your function to also be tested. The function is pretty trivial, so fine either way. However, I wanted to highlight that constants can be mocked just like functions)

```
monkeypatch.setattr(
        "breadbox.io.hdf5_utils.MAX_HDF5_READ_IN_BYTES", 10
    )
```